python/cuml/test/test_preprocessing.py

```diff
@@ -160,8 +160,6 @@ def test_standard_scaler_sparse(failure_logger,
 @pytest.mark.parametrize("axis", [0, 1])
 @pytest.mark.parametrize("with_mean", [True, False])
 @pytest.mark.parametrize("with_std", [True, False])
-# FIXME: ignore warnings from cuml and sklearn about scaling issues
-# issue: https://github.com/rapidsai/cuml/issues/4203
```

Comment:
Instead of removing the comment, can you add an explanation for filtering the warning instead?